docs/src/rsh/consensus/index.md

```diff
@@ -562,12 +562,12 @@ bidders.member(Alice); // false
 ```
 
 
-A `{!rsh} Set` is another container for linear state. It is simply a type alias of `{!rsh} Map(Null)`;
-it is only useful for tracking `{!rsh} Address`es. Because a `{!rsh} Set` is internally a `{!rsh} Map`, it may
-only be constructed in a consensus step.
+A `{!rsh} Set` is another container for linear state. It is simply a type alias of `{!rsh} Map(Null)`; it is only useful for tracking `{!rsh} Address`es. Because a `{!rsh} Set` is internally a `{!rsh} Map`, it may only be constructed in a consensus step.
```

Comment:
What is changed on this line? One thing that I aspire to in the docs is semantic linefeeds --- https://rhodesmill.org/brandon/2012/one-sentence-per-line/ --- and this change makes it worse